### DISCUSSION TITLE
suggestions for free deployment

### DISCUSSION THREAD

**Discussion by teranixbq** [community/community#143840] [Programming Help] [answered]
### Body

Any suggestions on where to deploy with Golang for free?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by extenv** [comment, marked as answer]
You can try deploying with koyeb.com

and see https://www.koyeb.com/pricing
![WhatsApp Image 2024-11-07 at 15 41 19](https://github.com/user-attachments/assets/acbb3698-2321-4140-b5f0-2d7f9cdec4c8)
